code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,347 @@
+      30px 0px transparent, 
+      30px 10px transparent, 
+      20px 20px transparent, 
+      10px 30px transparent, 
+      0px 30px transparent, 
+      -10px 30px transparent, 
+      -20px 20px transparent, 
+      -30px 10px #ffffff56, 
+      -30px 0px #ffffff56, 
+      -30px -10px #ffffff56, 
+      -20px -20px #ffffff56,
+      -10px -30px #ffffff56;
+  }
+  75% {
+    box-shadow: 
+      0px -30px #ffffff56, 
+      10px -30px #ffffff56, 
+      20px -20px #ffffff56, 
+      30px -10px #ffffff56, 
+      30px 0px transparent, 
+      30px 10px transparent, 
+      20px 20px transparent, 
+      10px 30px transparent, 
+      0px 30px transparent, 
+      -10px 30px transparent, 
+      -20px 20px transparent, 
+      -30px 10px transparent, 
+      -30px 0px #ffffff56, 
+      -30px -10px #ffffff56, 
+      -20px -20px #ffffff56,
+      -10px -30px #ffffff56;
+  }
+  81.25% {
+    box-shadow: 
+      0px -30px #ffffff56, 
+      10px -30px #ffffff56, 
+      20px -20px #ffffff56, 
+      30px -10px #ffffff56, 
+      30px 0px #ffffff56, 
+      30px 10px transparent, 
+      20px 20px transparent, 
+      10px 30px transparent, 
+      0px 30px transparent, 
+      -10px 30px transparent, 
+      -20px 20px transparent, 
+      -30px 10px transparent, 
+      -30px 0px transparent, 
+      -30px -10px #ffffff56, 
+      -20px -20px #ffffff56,
+      -10px -30px #ffffff56;
+  }
+  87.5% {
+    box-shadow: 
+      0px -30px #ffffff56, 
+      10px -30px #ffffff56, 
+      20px -20px #ffffff56, 
+      30px -10px #ffffff56, 
+      30px 0px #ffffff56, 
+      30px 10px #ffffff56, 
+      20px 20px transparent, 
+      10px 30px transparent, 
+      0px 30px transparent, 
+      -10px 30px transparent, 
+      -20px 20px transparent, 
+      -30px 10px transparent, 
+      -30px 0px transparent, 
+      -30px -10px transparent, 
+      -20px -20px #ffffff56,
+      -10px -30px #ffffff56;
+  }
+  93.75% {
+    box-shadow: 
+      0px -30px #ffffff56, 
+      10px -30px #ffffff56, 
+      20px -20px #ffffff56, 
+      30px -10px #ffffff56, 
+      30px 0px #ffffff56, 
+      30px 10px #ffffff56, 
+      20px 20px #ffffff56, 
+      10px 30px transparent, 
+      0px 30px transparent, 
+      -10px 30px transparent, 
+      -20px 20px transparent, 
+      -30px 10px transparent, 
+      -30px 0px transparent, 
+      -30px -10px transparent, 
+      -20px -20px transparent,
+      -10px -30px #ffffff56;
+  }
+  100% {
+    box-shadow: 
+      0px -30px #ffffff56, 
+      10px -30px #ffffff56, 
+      20px -20px #ffffff56, 
+      30px -10px #ffffff56, 
+      30px 0px #ffffff56, 
+      30px 10px #ffffff56, 
+      20px 20px #ffffff56, 
+      10px 30px #ffffff56, 
+      0px 30px transparent, 
+      -10px 30px transparent, 
+      -20px 20px transparent, 
+      -30px 10px transparent, 
+      -30px 0px transparent, 
+      -30px -10px transparent, 
+      -20px -20px transparent,
+      -10px -30px transparent;
```

Comment:
What an effort you've done with the box shadow here! :D Loadingspinner is on point with your theme!